From d279ffa49284b5e5f787f76edbe8c52226534a64 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Wed, 08 Jan 2014 06:29:04 +0000
Subject: Remove redundant declaration of FontFileStartListFonts()

Fixes gcc warning:
catalogue.c:336:1: warning: redundant redeclaration of
 'FontFileStartListFonts' [-Wredundant-decls]
In file included from ../../include/X11/fonts/fntfilst.h:40:0,
                 from catalogue.c:32:
../../include/X11/fonts/fntfil.h:92:12: note: previous declaration
 of 'FontFileStartListFonts' was here

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
Reviewed-by: Jasper St. Pierre <jstpierre@mecheye.net>
---
diff --git a/src/fontfile/catalogue.c b/src/fontfile/catalogue.c
index 6b6fc09..6230930 100644
--- a/src/fontfile/catalogue.c
+++ b/src/fontfile/catalogue.c
@@ -332,11 +332,6 @@ CatalogueListFonts (pointer client, FontPathElementPtr fpe, char *pat,
     return Successful;
 }
 
-int
-FontFileStartListFonts(pointer client, FontPathElementPtr fpe,
-		       char *pat, int len, int max,
-		       pointer *privatep, int mark_aliases);
-
 typedef struct _LFWIData {
     pointer		*privates;
     int			current;
--
cgit v0.9.0.2-2-gbebe
